components/Card/index.js

```diff
@@ -66,9 +91,26 @@ export default function Card({ initialItems, items }) {
           })}
         </ul>
       )}
+
       <ul style={{ listStyle: "none" }}>
-        {selectedItems.map((selectedItem) => {
-          return <li key={selectedItem.name}>{selectedItem.name} </li>;
+        {selectedItems.map((selectedItem, index) => {
+          return (
+            <li key={selectedItem.name}>
+              {" "}
```

Comment:
this can be deleted

Reply:
i hope i get it right and you meant the space ? and in this case  the index in the brakets.

Reply:
Yeah I think it's about Line 99